dvc/exceptions.py

```diff
@@ -47,7 +47,8 @@ def __init__(self, output, stages):
         )
         msg = (
             "file/directory '{}' is specified as an output in more than one "
-            "stage: {}"
+            "DVC-file, which is not allowed: {}\n"
```

Comment:
Hmmm. We use the term "stage" in the console output a lot so I see no problem with just:

```suggestion
            "stage: {}\n"
```

The "which is not allowed" part is a little confusing in this sentence, maybe in the next line?

Reply:
See https://github.com/iterative/dvc/pull/2941#discussion_r356996806.